### PR TITLE
Clean up disambiguation articles; fix links to the article about difficulties

### PR DESCRIPTION
**Pull request by TicClick**

### CODE REVIEW

File: wiki/Disambiguation/Mod/en.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 **Mod** can have multiple meanings:
 
-- A moderator, such as an [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member.
-- [The feedback](/wiki/Modding) given by a person who looks over a beatmap.
-- A [game modifier](/wiki/Game_modifier).
+- A moderator, such as a [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or [GMT](/wiki/People/The_Team/Global_Moderation_Team) member
```
Comment (MilkyIQ) [inline wiki/Disambiguation/Mod/en.md:5]: dont see why this change was neccessary? English uses a vs an based off the pronounciation of the acronym itself and not the words in the acronym afaik.

File: wiki/Disambiguation/Offset/en.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
 
 ## Timing settings
 
-- An [uninherited ("red") offset](/wiki/Beatmapping/Offset), which denotes either the song's first downbeat, or a start of a new timing section.
-- [Online offset](/wiki/Beatmap/Online_offset), a timing change applied by [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) to ranked beatmaps with incorrect uninherited offset in order to adjust it.
-- [Local ("offline") offset](/wiki/Glossary#local-song-offset), a per-beatmap setting that shifts the objects in time during gameplay.
-- [Universal offset](/wiki/Universal_offset), the osu! client setting that globally affects the timing of beatmaps.
+- An [uninherited ("red") offset](/wiki/Beatmapping/Offset), which denotes either the song's first downbeat, or a start of a new timing section
```
Comment (MilkyIQ) [inline wiki/Disambiguation/Offset/en.md:7]:
```suggestion
- An [uninherited ("red") offset](/wiki/Beatmapping/Offset), which denotes either the song's first downbeat, or the start of a new timing section
```

File: wiki/Disambiguation/Mod/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Disambiguation/Mod/en.md:5]: I know, it's... it's just that I've thought for a second that it reads as `'næt`. duh

Comment (MilkyIQ) [reply wiki/Disambiguation/Mod/en.md]: huh, i was pronouncing it "N-A-T". so i actually dont know which one is correct here. either way it doesnt matter much

**Review by MilkyIQ — APPROVED** [review]
links work fine afaict

**Review by TPGPL — APPROVED** [review]
